File: Open3D/03_transformaciones/cpp/main.cpp
#include <open3d/Open3D.h>
int main(){ auto pc=open3d::io::CreatePointCloudFromFile("input.ply"); if(!pc) return 1; Eigen::Matrix4d T=Eigen::Matrix4d::Identity(); T(0,3)=0.2; T(2,3)=0.1; pc->Transform(T); open3d::io::WritePointCloud("transformed.ply",*pc); }

File: Open3D/03_transformaciones/python/main.py
import open3d as o3d, numpy as np
pc=o3d.io.read_point_cloud("input.ply"); T=np.eye(4); T[:3,3]=[0.2,0.0,0.1]; pc.transform(T); o3d.io.write_point_cloud("transformed.ply",pc)
